data/geography/canadian_municipalities.json

```diff
@@ -0,0 +1,606 @@
+{
+    "description": "Top 100 Canadian municipalities by 2011 population",
+    "source": "https:\/\/en.wikipedia.org\/wiki\/List_of_the_100_largest_municipalities_in_Canada_by_population",
```

Comment:
I just wrote a quick script to parse the table on wikipedia and used a JSON encoding function that does it automatically. It's apparently in the JSON spec to escape the / char (http://www.json.org/) but it's not required. Since it doesn't fit in with everything else I'll just remove the escaping.